packages/cli/src/lib/project/project.ts

```diff
@@ -88,12 +93,24 @@ export class Project {
         archive.directory(this.resourcePath, false);
         archive.finalize();
       });
+      if (cachedManifest) {
```

Comment:
Could this block and lines 101-103 be moved to a `finally` block to remove repetition ?

Reply:
That was my first attempt, but because there's a `return` statement 3 lines down below, the finally clause doesn't work.
If you have another suggestion, be my guest tho, I ain't liking the repetition either.